src/Components/common/getUrlInputFile.js

```diff
@@ -0,0 +1,12 @@
+
+export const UrlInput = (e,setUrl,setFieldValue) =>{
+
+  const imageFile = e.target.files[0];
+  const imageUrl = new FileReader();
+  imageUrl.readAsDataURL(imageFile)
+  imageUrl.onload=(e)=>{
+    setFieldValue("image",e.target.result)
+    setUrl(e.target.result)
```

Comment:
this is the third responsability that it..., well, I do not know what it do. It sets somewhere the url of the result of an event. But it looks to be responsability of the TestimonialForm

Reply:
I'm sorry, I don't understand what you mean by this, or what do you mean with the responsibility of the form?